src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -567,10 +568,45 @@ export default class OncoprintControls extends React.Component<
             label: d.id,
         }));
     }
-    
+
+    @autobind
+    @action
+    onInputChange(input: string) {
```

Comment:
this isnt really idiomatic to how we've been doing things. the way we would do things is keep an observable for the input, change the input in `onInputChange`, then make `@computed get filteredTreatmentOptions()` return what you compute here.

Reply:
Thanks for your review. I have updated my PR.